release-notes/v1_10.md

```diff
@@ -334,13 +338,17 @@ Note: A debug adapter backend can update the location of breakpoints during a de
 
 ![column](images/1_10/column-breakpoints.gif)
 
-### Ability to select and start a launch using keyboard
+### Ability to Select and Start a Launch using Keyboard
 
-TODO@Michel
+The option to launch debug configurations using keyboard was introduced, as asked in [this request](https://github.com/Microsoft/vscode/issues/16613). It works similarly to running tasks, with an ability to launch from the Command Palette. The keyword `'debug '` or the option **Debug: Select and Start Debugging** from Command Palette is used to pull and debug any configuration from launch.json. 
+
+![launch](images/1_10/launch-keyboard.gif)
 
 ### Improved Exception experience
 
-TODO@Michel
+To give a better visibility for the exceptions occuring in debugging mode, user-friendly widget pops up now. This was raised [here](https://github.com/OmniSharp/omnisharp-vscode/issues/1007), and becomes very practical in cases when debugging stops at some point, so that user is not aware that an exception was thrown (especially if the exception was thrown in the library code).
```

Comment:
Remove 'user-friendly widget'.
A user-friendly is not objective it is up to the users to decide that. 'Widget' is not a term we surface to users.
Maybe use the term 'exception dialog'.
Mention also that one of the motivations for this is to make it easier to the user to read the exception details